emergence_lib/src/hive_mind.rs

```diff
@@ -39,11 +39,15 @@ pub enum HiveMindAction {
 /// Enumerates the actions a developer can take.
 #[derive(Actionlike, Clone, Copy, PartialEq, Eq, Debug)]
 pub enum DevAction {
+    /// Toggle the overall developer mode setting
+    ToggleDevMode,
     // TODO: make debug labels
     /// Toggle tilemap labels tools
-    ShowDebugLabels,
+    ToggleTileLabels,
     /// Toggle rendering info
-    ShowInfoText,
+    ToggleInfoText,
+    /// Toggle the inspector
+    ToggleInspector,
 }
 
 // TODO: rework this to use LWIM conventions for mapping controls
```

Comment:
Hm, I think this TODO is now wrapped up by your work

Reply:
Got it